src/utils/HTMLCollection.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * Class provides interface for HTLMCollection. Used in getElementsByTagName etc.
+ *
+ * @see https://developer.mozilla.org/en-US/docs/Web/API/HTMLCollection
+ */
+export default class HTMLCollection extends Array {
+    constructor(parent, selector) {
+        super();
+        this._parent = parent;
+        this._selector = selector;
+        this._fillChildren(this._parent);
+    }
+
+    _fillChildren(parent) {
```

Comment:
Yep, I know that. This kind of next step. I required HTMLCollection as interface only (especially `item()` method). Hope later I will make it live one.